recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,2 @@
 MACOSX_DEPLOYMENT_TARGET:   # [osx]
-- '10.12'                   # [osx]
```

Comment:
@isuruf and @mingwandroid is the failure here related to 10.10? Should we force 10.12 for this build?

Reply:
I believe macOS SDK 10.10 is what AD targets. We don't have a conda_build_config.yaml for Qt (or at least didn't for 5.9).

Reply:
Thanks. I'm quite lost in all the commit here. I'm also trying from a "fresh" PR based on AR but there I'm getting many undefined reference to `dbus` stuff, like:

```
lib/libQt5WebEngineCore.so: undefined reference to `dbus_message_set_serial@LIBDBUS_1_3'
```

In the past we had `dbus` from `yum_requirements.txt` but now we have a package in `host`. Not sure how to fix that.

Reply:
I already fixed that in

https://github.com/conda-forge/qt-feedstock/pull/88/commits/e367b5dbc31d392e726d0d11d4e73b5ef157a4e0

Reply:
I'm fixing run_exports in dbus so we can try to keep the recipe as close as possible to AR.